fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindRelation.java

```diff
@@ -136,6 +136,9 @@ private LogicalPlan bindWithCurrentDb(CascadesContext cascadesContext, UnboundRe
         TableIf table = null;
         if (cascadesContext.getTables() != null) {
             table = cascadesContext.getTableByName(tableName);
+            if (ConnectContext.get().getSessionVariable().isPlayNereidsDump() && table == null) {
+                throw new AnalysisException("Can not find table:" + tableName);
+            }
```

Comment:
if getTableByName only used for minidump, please throw exception in it self and change the method name to reflect it used for minidump

Reply:
done